rllib/policy/dynamic_tf_policy_v2.py

```diff
@@ -473,11 +474,16 @@ def _init_input_dict_and_dummy_batch(
             )
         else:
             # Placeholder for (sampling steps) timestep (int).
-            timestep = tf1.placeholder_with_default(
-                tf.zeros((), dtype=tf.int64), (), name="timestep"
-            )
+            if not hasattr(self, "timestep_placeholder"):
```

Comment:
This is the main point of this fix. When loading a checkpoint twice, we try to set these placeholder twice.

Reply:
It does not work like tf.Variable(reuse=True), you can't create a placeholder with the same name twice